src/ScottPlot/Plottable/Circle.cs

```diff
@@ -0,0 +1,133 @@
+﻿using ScottPlot.Drawing;
+using System;
+using System.ComponentModel;
+using System.Drawing;
+using System.Data;
+
+namespace ScottPlot.Plottable
+{
+    /// <summary>
+    /// Display a text label at an X/Y position in coordinate space
+    /// </summary>
+    public class Circle : IPlottable
+    {
+        // data
+        public double X;
+        public double Y;
+        public string Label;
+        public double Radius;
+        public bool AllowEllipse;
```

Comment:
I modified it so they're obligate circles with a position in coordinate units and a size in pixel units. Since pixels are always square, these shapes are now always circles. 

It's not super obvious to me whether size should be coordinate units or pixel units. It probably depends on what type of data the user intends to plot with this plot type. Currently the pixel size behaves similarly to markers.